### PR TITLE
Add Purdue LUG mirror

### PR DESCRIPTION
**Pull request by zekesonxx**
Hi, I'm one of the maintainers of the Purdue LUG mirror.
We've added your packages to our mirror, accessible at https://plug-mirror.rcac.purdue.edu/termux/.
Our server has a 10g pipe to the internet, and can realistically push that much traffic.

The packages are currently updating from `rsync@grimler.se::termux`, four times daily.
We have a privacy policy, available on our mirror at https://plug-mirror.rcac.purdue.edu/info.html.

Hoping we can get on your official mirror list.

Thanks!

### CODE REVIEW

File: packages/termux-tools/pkg [hunk 1]
```diff
@@ -80,6 +80,7 @@ select_mirror() {
 	mirrors[11]="https://termux.librehat.com/apt/termux-main"
 	mirrors[10]="https://mirror.mwt.me/termux/main"
 	mirrors[ 9]="https://termux.sahilister.in/apt/termux-main"
+	mirrors[ 8]="https://plug-mirror.rcac.purdue.edu/termux/termux-main"
```
Comment (thunder-coding) [inline packages/termux-tools/pkg:83]: 16 + 15 + ... + 8 != 100. You need to adjust these values such that their sum is 100.

**Review by thunder-coding — CHANGES_REQUESTED** [review]
Also you need to add your mirror to `termux-change-repo` too! In case any help needed, feel free to ask.

A version bump is also needed

File: packages/termux-tools/pkg (hunk 1 above)
Comment (zekesonxx) [reply packages/termux-tools/pkg:83]: I'll be honest and say I don't understand how to properly adjust them then. Could you calculate what values I should set them to? I think I understand how to adjust the other file and version bump it.

Comment (Grimler91) [reply packages/termux-tools/pkg:83]: Thanks for setting up/maintaining the mirror! Please change the weights in the list to 16, 14, 13, 12, 11, 10, 9, 8, 7

**Comment by zekesonxx** [comment]
How's that look?

**Review by buttaface — APPROVED** [review]

**Review by thunder-coding — APPROVED** [review]
LGTM

**Pull request merged**